tensorflow_datasets/structured/movielens.py

```diff
@@ -48,22 +48,49 @@
 """
 
 _DESCRIPTION = """
-This dataset describes 5-star rating from MovieLens,
-a movie recommendation service.
-Users were selected at random for inclusion.
-There are 5 versions included: 25m, latest-small, 100k, 1m, 20m.
-In all datasets, the movies data and ratings data are joined on "movieId".
-The 25m dataset, latest-small dataset, and 1m dataset have the same schema
-and they do not include demographic data.
-The 25m dataset is the latest stable version of the MovieLens dataset.
-It is recommended for research purposes.
-The latest-small dataset is a small dataset with
-the same schema as 25m and 20m.
-It is updated over time by GroupLens and its checksum might need updating.
-The 20m dataset is one of the most used MovieLens datasets in academic papers
+This dataset contains a set of movie ratings from the MovieLens website, a movie
+recommendation service. This dataset was collected and maintained by [GroupLens]
+(https://grouplens.org/), a research group at the University of Minnesota. There
+are 5 versions included: "25m", "latest-small", "100k", "1m", "20m". In all
+datasets, the movies data and ratings data are joined on "movieId". The 25m
+dataset, latest-small dataset, and 20m dataset contain only movie data and
+rating data. The 1m dataset and 100k dataset contain demographic data in
+addition to movie and rating data.
+
+- "25m": This is the latest stable version of the MovieLens dataset. It is
+recommended for research purposes.
+- "latest-small": This is a small subset of the latest version of the MovieLens
+dataset. It is updated over time by GroupLens and its checksum might need
```

Comment:
I am not sure we need to mention the checksum here: this is mostly a note to TFDS developers and not to users. What do you think?

Reply:
I agree. It is more important for the developers to know this than for the users to know. I have removed this from the documentation string added it as a comment at line 184 in dfe803aa3e5f9ddba2d9e26214c612f095ea1b2a.